src/pages/reset-password/index.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Formik, Form } from 'formik'
+import Layout from 'components/layout/layout'
+import { localize, WithIntl } from 'components/localization'
+import { Container, SEO } from 'components/containers'
+import { Header, Text } from 'components/elements'
+import { Input, Button } from 'components/form'
+import validation from 'common/validation'
+import { trimSpaces } from 'common/utility'
+import { BinarySocketBase } from 'common/websocket/socket_base'
+import Login from 'common/login'
+
+type EmailType = { email: string }
+
+type ErrorType = { email?: string }
```

Comment:
You can use Utility types to manage/reuse declared types to fit on specifications. 

`
type ErrorType = Partial<EmailType>
`